File: app/src/main/java/com/arejas/dashboardofthings/presentation/ui/notifications/ToastHelper.java
package com.arejas.dashboardofthings.presentation.ui.notifications;

import android.widget.Toast;

import com.arejas.dashboardofthings.DotApplication;

public class ToastHelper {

    private static Toast mToast;

    public static void showToast(String message) {
        if (mToast != null) mToast.cancel();
        mToast = Toast.makeText(DotApplication.getContext(), message, Toast.LENGTH_SHORT);
        mToast.show();
    }

}

File: app/src/main/java/com/arejas/dashboardofthings/presentation/ui/notifications/ToastHelper.kt
package com.arejas.dashboardofthings.presentation.ui.notifications

import android.widget.Toast

import com.arejas.dashboardofthings.DotApplication

object ToastHelper {

    private var mToast: Toast? = null

    fun showToast(message: String) {
        if (mToast != null) mToast!!.cancel()
        mToast = Toast.makeText(DotApplication.context, message, Toast.LENGTH_LONG)
        mToast!!.show()
    }

}
